launcher/nvdaLauncher.nsi

```diff
@@ -72,7 +72,10 @@ Banner::show /nounload
 BringToFront
 
 setOutPath "$PLUGINSDIR"
-;Play NVDA logo sound
+;Play NVDA logo sound, unless started with the --minimal command line parameter
+${GetParameters} $0
+${GetOptions} $0 "--minimal" $1
+IfErrors 0 +3

```

Comment:
If the error flag is set, jump to first param (which is `0`), otherwise jump to second param (which is `+3`). These are treated as offsets from the current line. `+3` will jump to `Call PlaySound`, but the sound file path will not have been pushed to the stack so likely fails. Instead perhaps `ifError 0 +1` then the next line should be `Call PlayLogo` and a new function added:

```NSIS
File "..\miscDeps\launcher\nvda_logo.wav"
Push "$PLUGINSDIR\nvda_logo.wav"
Call PlaySound
```

Advice in https://nsis.sourceforge.io/Docs/Chapter2.html#tut-working-with-scripts suggests using `logiclib` rather than iferrors / goto. 

https://nsis.sourceforge.io/Docs/Chapter4.html#iferrors
https://nsis.sourceforge.io/Docs/Chapter4.html#jumps

Reply:
Refactored as 5ebaa614f

Actually, the +3 jump did not fail during my testing… Hence I thought the Goto offset was relative to the next line rather than the current one.
Anyway, using LogicLib now.